client/app.jsx

```diff
@@ -1,8 +1,52 @@
 import React from 'react';
-import Home from './pages/home';
+import Welcome from './pages/Welcome';
+import Login from './pages/Login';
+import Authorize from './src/Authorize';
+import parseRoute from './lib/parse-route';
+import '../server/public/login.css';
```

Comment:
Instead of importing from `server/public`, move this css file next to the login component and import it there.

Under normal circumstances, you just link your css in your HTML document, but if the CSS is for a specific component, put the file near the component